libraries/botbuilder-dialogs/tests/test_skill_dialog.py

```diff
@@ -217,8 +220,223 @@ async def test_should_throw_on_post_failure(self):
         with self.assertRaises(Exception):
             await client.send_activity("irrelevant")
 
-    def _create_skill_dialog_options(
-        self, conversation_state: ConversationState, skill_client: BotFrameworkClient
+    async def test_should_intercept_oauth_cards_for_sso(self):
+        connection_name = "connectionName"
+        first_response = ExpectedReplies(
+            activities=[
+                SkillDialogTests.create_oauth_card_attachment_activity("https://test")
+            ]
+        )
+
+        sequence = 0
+
+        async def post_return():
+            nonlocal sequence
+            if sequence == 0:
+                result = InvokeResponse(body=first_response, status=200)

```

Comment:
In C# and js we have constants for status. HttpStatusCodes. Thoughts around adopting something similar? Have you considered this already?

Reply:
There is.  You don't like magic numbers, Carlos? ;-)